static/app/views/releases/components/header.tsx

```diff
@@ -86,6 +88,7 @@ function Header({router, hasV2ReleaseUIEnabled = false, organization}: Props) {
                     delay={SLOW_TOOLTIP_DELAY}
                   >
                     {label}
+                    {badge}
```

Comment:
if `badge` does not exist, is this safe?

Reply:
Yup - just doesn't render anything